README.md

```diff
@@ -111,11 +113,8 @@ A JS configuration file may internally `require` one or more npm packages as a w
 
 MIT © Igor Shubovych
 
-[travis-badge]: https://img.shields.io/travis/igorshubovych/markdownlint-cli/master.svg?label=linux
-[travis-url]: https://travis-ci.org/igorshubovych/markdownlint-cli
-
-[appveyor-badge]: https://img.shields.io/appveyor/ci/igorshubovych/markdownlint-cli/master.svg?label=windows
-[appveyor-url]: https://ci.appveyor.com/project/igorshubovych/markdownlint-cli
+[actions-badge]: https://github.com/igorshubovych/markdownlint-cli/workflows/Test/badge.svg
```

Comment:
Please use the versions that lock to "master" branch.

Reply:
Looks like it uses "Default branch" if you don't specify it, but I pinned it to master anyway